### PR TITLE
Fix error in page_foreground and cleanup

### PR DESCRIPTION
**Pull request by titodalcanton**
Fixes this error reported by Sunil:
```
Traceback (most recent call last):
  File "/home/sunil.choudhary/39_max/bin/pycbc_page_foreground", line 183, in <module>
    fortrigs.to_coinc_xml_object(args.output_file)
  File "/home/sunil.choudhary/39_max/lib/python3.9/site-packages/pycbc/io/hdf.py", line 861, in to_coinc_xml_object
    proc_id = ligolw_process.register_to_xmldoc(outdoc, 'pycbc',
  File "/home/sunil.choudhary/39_max/lib/python3.9/site-packages/ligo/lw/utils/process.py", line 100, in register_to_xmldoc
    process = proctable.RowType.initialized(program = program, process_id = proctable.get_next_id(), **kwargs)
  File "/home/sunil.choudhary/39_max/lib/python3.9/site-packages/ligo/lw/lsctables.py", line 586, in initialized
    cvs_entry_time = lal.UTCToGPS(time.strptime(cvs_entry_time, "%Y-%m-%d %H:%M:%S +0000")) if cvs_entry_time is not None else None,
  File "/cvmfs/oasis.opensciencegrid.org/ligo/sw/conda/envs/igwn-py39/lib/python3.9/_strptime.py", line 562, in _strptime_time
    tt = _strptime(data_string, format)[0]
  File "/cvmfs/oasis.opensciencegrid.org/ligo/sw/conda/envs/igwn-py39/lib/python3.9/_strptime.py", line 349, in _strptime
    raise ValueError("time data %r does not match format %r" %
ValueError: time data '' does not match format '%Y-%m-%d %H:%M:%S +0000'
```
and cleans up the executable script.

Not tested.

### CODE REVIEW

File: pycbc/io/hdf.py [hunk 1]
```diff
@@ -857,11 +857,13 @@ def to_coinc_xml_object(self, file_name):
         outdoc = ligolw.Document()
         outdoc.appendChild(ligolw.LIGO_LW())
 
-        ifos = list(self.sngl_files.keys())
-        proc_id = ligolw_process.register_to_xmldoc(outdoc, 'pycbc',
-                     {}, instruments=ifos, comment='', version=pycbc_version.git_hash,
-                     cvs_repository='pycbc/'+pycbc_version.git_branch,
-                     cvs_entry_time=pycbc_version.date).process_id
+        ifos = sorted(self.sngl_files)
+        proc_table = create_process_table(
```
Comment (GarethCabournDavies) [inline pycbc/io/hdf.py:861]: has create_process_table been imported (I guess this will show up when tested)

**Review by GarethCabournDavies — COMMENTED** [review]
Everything looks sensible to me, apart from one question, which will show up in testing anyway.

File: pycbc/io/hdf.py (hunk 1 above)
Comment (titodalcanton) [reply pycbc/io/hdf.py:863]: :picard:

Comment (GarethCabournDavies) [reply pycbc/io/hdf.py:863]: :thanks: (I like that this showed up properly in the gwastro pycbc-github channel)

**Comment by titodalcanton** [comment]
This change will imply that the version field of the LIGOLW document will now report `pycbc_version.version` instead of `pycbc_version.git_hash`, but I think that is fine, and it will be consistent with other codes.

**Review by GarethCabournDavies — APPROVED** [review]

**Pull request merged**